Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,271 @@
+            true
+        ),
+        new Wishable(
+            $item[none],
+            $effect[Painted-On Bikini],
+            HTMLGenerateSpanFont("+100 sleaze damage", "purple"),
+            get_property_int("zeppelinProtestors") < 80,
+            true
+        ),
+        new Wishable(
+            $item[glark cable],
+            $effect[none],
+            "",
+            __quest_state["Level 11 Ron"].mafia_internal_step < 5,
+            locationAvailable($location[The Red Zeppelin])
+        ),
+        new Wishable(
+            $item[short writ of habeas corpus],
+            $effect[none],
+            "",
+            !__quest_state["Level 11 Hidden City"].finished,
+            locationAvailable($location[The Hidden Park])
+        ),
+        new Wishable(
+            $item[lion oil],
+            $effect[none],
+            "",
+            $item[mega gem].available_amount() < 1 &&
+                $item[lion oil].available_amount() < 1,
+            locationAvailable($location[Whitey's Grove])
+        ),
+        new Wishable(
+            $item[bird rib],
+            $effect[none],
+            "",
+            $item[mega gem].available_amount() < 1 &&
+                $item[bird rib].available_amount() < 1,
+            locationAvailable($location[Whitey's Grove])
+        ),
+        new Wishable(
+            $item[drum machine],
+            $effect[none],
+            "",
+            get_property_int("desertExploration") < 100 &&
+                $item[drum machine].available_amount() < 1,
+            locationAvailable($location[The Oasis])
+        ),
+        new Wishable(
+            $item[green smoke bomb],
+            $effect[none],
+            "",
+            __quest_state["Level 12"].state_string["Side seemingly fighting for"] != "hippy",
+            !__quest_state["Level 12"].finished &&
+                locationAvailable($location[The Battlefield (Frat Uniform)])
+        ),
+        new Wishable(
+            $item[star chart],
+            $effect[none],
+            "",
+            !__quest_state["Level 13"].state_boolean["Richard's star key used"] &&
+                $item[Richard's star key].available_amount() < 1 &&
+                $item[star chart].available_amount() < 1,
+            locationAvailable($location[The Hole In The Sky])
+        ),
+        new Wishable(
+            $item[none],
+            $effect[Frosty],
+            "init/item/meat for 8-bit",
+            !__quest_state["Level 13"].state_boolean["digital key used"] &&
+                $item[digital key].available_amount() < 1 &&
+                get_property("8BitScore") < 10000,
+            true
+        ),
+        new Wishable(
+            $item[lowercase N],
+            $effect[none],
+            "",
+            !__quest_state["Level 13"].state_boolean["king waiting to be freed"] &&
+                $item[wand of nagamar].available_amount() < 1 &&
+                $item[lowercase N].available_amount() < 1,
```

Comment:
similarly, only recommend this if the user has no lowercase N but -does- have a heavy D, ruby W, and metallic A; sometimes you don't have em all!